src/server/scripts/Commands/cs_server.cpp

```diff
@@ -132,7 +132,10 @@ class server_commandscript : public CommandScript
     // Display the 'Message of the day' for the realm
     static bool HandleServerMotdCommand(ChatHandler* handler, char const* /*args*/)
     {
-        handler->PSendSysMessage(LANG_MOTD_CURRENT, Motd::GetMotd());
+        if (handler->IsHumanReadable())
```

Comment:
Alternative would be to switch all command output to fmtlib named arguments (see http://fmtlib.net/latest/index.html), then you would have access to arg names